addons/repair/CfgVehicles.hpp

```diff
@@ -480,7 +482,22 @@ class CfgVehicles {
         GVAR(canRepair) = 0;
     };
 
-    class Truck_03_base_F;
+    class Truck_F: Car_F {
+        class HitPoints: HitPoints {
+            class HitLBWheel;
+            class HitRBWheel;
+        };
+    };
+    class Truck_03_base_F: Truck_F {
+        class HitPoints: HitPoints {
+            class HitLBWheel: HitLBWheel {
```

Comment:
`bin\config.bin/CfgVehicles/Car_F/HitPoints/HitLBWheel` NOT Truck_F HitLBWheel!

Reply:
it's still Car_F, checked in config dumper, no worries

Reply:
I would think it should be 
```
    class Car_F: Car {
        class HitPoints {
            class HitLBWheel;
            class HitRBWheel;
        };
    };
    class Truck_F: Car_F {
        class HitPoints: HitPoints {};
    };
    class Truck_03_base_F: Truck_F {
        class HitPoints: HitPoints {
            class HitLBWheel: HitLBWheel {
```
but this works fine I think
`(configProperties [configfile >> "CfgVehicles" >> "Truck_F" >> "HitPoints", "isClass _x", false]) apply {configName _x}` = []